src/pages/FrontendMentor/hubble/index.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import {
+  MainContainer,
+  LColumn,
+  LogoBox,
+  Rows,
+  HeaderText,
+  BodyText,
+  ButtonContainer,
+  SocialIcons,
+  IconRows,
+} from './hubble.styled'
+//import HubbleCard from '../../../components/HubbleCard'
+import comp from '../../../assets/hubble/illustration-mockups.svg'
+import logo from '../../../assets/hubble/logo.svg'
+
+const Hubble = () => {
+  return (
+    <>
+      <MainContainer>
+        <Rows>
+          <LogoBox>
+            <img src={logo} />
+          </LogoBox>
+        </Rows>
+        <Rows>
+          <LColumn>
+            <img src={comp} />
```

Comment:
hmmm I can't tell what image 'comp' is referring to at first glance, took a while to figure out, perhaps can use a clearer name to represent the image? could try interfaceDesign or layoutImages etc